router/auth.go

```diff
@@ -84,6 +84,42 @@ func authPassThrough(log *log.Logger, w http.ResponseWriter, req *http.Request,
 
 func authInjectECS(log *log.Logger, w http.ResponseWriter, req *http.Request, target *targetPassThroughAuth) bool {
 	req.SetBasicAuth(target.config.Username, target.config.Password)
+	var didWhat string
+	var toWhat string
+	actionUrl := strings.Split(req.URL.String(), "/") // Example: "ecs/ACCESS/FWVERSION/" or "ecs/sam/ForceSim1/"
+	if len(actionUrl) == 5 {
+		switch actionUrl[2] {
+		case "ACCESS":
```

Comment:
One could 
1. Completely remove the `else`
2. Set the `didWhat` and `toWhat` to their default values (`req.URL.String()`) before the `if`. (Replacing the `var` declarations with `:=` type assignments)
3. Move the `didWhat = actionUrl[3]` statement to right before the `switch` statement within the `if` and
4. Omit the `default` case